app/players/dwarves/dwarf.py

```diff
@@ -0,0 +1,10 @@
+from app.players.player import Player
+
+
+class Dwarf(Player):
```

Comment:
You also need to make it clear that this is an abstract class. `class Dwarf(Player, ABC):`

Reply:
thx, fixed